.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Is there a better base image we could use?

In the long term we should roll our own Docker image imho but that doesn't have to be a this-PR problem.

Reply:
This is a VM template, not a Docker image, as I understand it. The options for that field are only Ubuntu 18.04, Ubuntu 16.04, Windows, and macOS.

Reply:
Ah ok. From the docs it sounds like individual steps are docker containers though?

Reply:
At the very least we have the option to wrap it in a container.